src/deployment/azuredeploy.json

```diff
@@ -583,6 +538,46 @@
                 }
             ]
         },
+
+        {
+            "apiVersion": "2020-02-02-preview",
+            "name": "[parameters('name')]",
+            "type": "microsoft.insights/components",
+            "location": "[resourceGroup().location]",
+            "kind": "",
+            "properties": {
+                "ApplicationId": "[parameters('name')]",
+                "Application_Type": "other",
+                "RetentionInDays": "[variables('log_retention')]",
+                "WorkspaceResourceId" : "[resourceId('Microsoft.OperationalInsights/workspaces', parameters('name'))]"
```

Comment:
Should this be concatenating `variables('monitorAccountName')`? Might want to double check all these final uses.

Reply:
Ah yes thanks for catching.

And yes, I'm redeploying the updated instance to make sure everything still works properly